src/tokens/testnet/defaultGaugeList.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
```

Comment:
this is now a [Record<string,obj>]. plz match the default token list interface, add linting / checking to these files as well